### ISSUE TITLE
TX history cropped (Win)

### ISSUE THREAD

**Issue by allienworks** [particl/particl-desktop#821]
_source: https://github.com/particl/particl-desktop/issues/753#issuecomment-365299542_

- Windows bug
- RC4 Sandboxed version

![](https://user-images.githubusercontent.com/30150617/36157527-e3e2e4ec-10d1-11e8-976b-c7b93ec10c80.png)

**Comment by allienworks** [comment]
I investigated this today, thought it was bad setup of flex layout.. but unfortunately not (there's a plenty of space for TX amount available). I will look into it on my Win machine later.

**Comment by kewde** [comment]
@allienworks for what it's worth, test it on sandboxed-RC5, nothing changed on the Angular side of things.
Note: check how flexbox deals with resizing maybe?

**Comment by allienworks** [comment]
@kewde nah, `min-width` of the "cell" is defined. But I vaguely remember that we had some similiar issue already earlier.. I'll look into it later.

**Comment by allienworks** [comment]
Sorry, I can't reproduce this. I have issues with my Win machine :|
Can anyone else look into it? @gerlofvanek maybe?

**Comment by AllienWorks** [comment]
@imybb Do you still have this issue? I believe it has been resolved already.
